roles/jboss_ws/tasks/main.yml

```diff
@@ -74,34 +74,48 @@
   ignore_errors: yes
   when: 'jws_home is defined and jboss_ws'
 
-- name: check for jws cert
+- name: check for jws cert in product directory
   raw: ls /etc/pki/product/185.pem 2>/dev/null
-  register: jws_has_cert
+  register: jws_product
+  ignore_errors: yes
+  when: 'jboss_ws'
+
+- name: check for jws cert in product-default directory
+  raw: ls /etc/pki/product-default/185.pem 2>/dev/null
+  register: jws_product_default
+  ignore_errors: yes
+  when: 'jboss_ws'
+
+- name: combine jws certs
+  set_fact:
+    jws_has_certs: "{{
```

Comment:
Now that this is named jws_has_certs instead of jws_has_cert we might want to update the processing.

Reply:
Opps I think we should just rename it back to jws_has_cert. Because it will still only find one cert.

Reply:
That's totally fine but you're going to want to do a lot of running actual scans to make sure that it works as expected (scan a system that has a pem file in each directory/ a pem file in either directory/ etc.).

Reply:
Done. Kev informed me that there were multiple ways to detect jboss_ws. So I have ssh into a machine without jboss present and added /etc/pki/product-default/185.pem. Just tested and I got present for jboss_ws

Reply:
This definitely is better.  You will notice that we missed a lot of 69.pem before this PR.  Wow! Here is a comparison of 0.0.44 to this PR (edited a bit):
```
Systems in 44-large-network.json but not 1624-large-network.json: 

#### dhcp-10-12-211-111.hq.gsslab.rdu.redhat.com--00:50:56:9E:4F:3D - (system1, system2) ####
mismatch: redhat_certs (69.pem,185.pem;69.pem;)

#### dhcp-10-12-211-130.hq.gsslab.rdu.redhat.com--00:50:56:9E:A1:E9 - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp-10-12-211-33.hq.gsslab.rdu.redhat.com--00:50:56:9E:7A:2C - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp-10-12-211-93.hq.gsslab.rdu.redhat.com--00:50:56:9E:97:EB - (system1, system2) ####
mismatch: redhat_certs (201.pem,201.pem;69.pem;)

#### dhcp181-135.gsslab.rdu2.redhat.com--00:50:56:9E:02:E6 - (system1, system2) ####
mismatch: redhat_certs (183.pem,183.pem;69.pem;)

#### dhcp181-62.gsslab.rdu2.redhat.com--00:50:56:9E:CF:A6 - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp182-157.gsslab.rdu2.redhat.com--00:50:56:9E:C8:C6 - (system1, system2) ####
mismatch: redhat_certs (183.pem,183.pem;69.pem;)

#### dhcp182-33.gsslab.rdu2.redhat.com--00:50:56:9E:39:EB - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp182-37.gsslab.rdu2.redhat.com--00:50:56:9E:3A:1F - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)
```

Reply:
🥇 Nice catch by dan to find the product-default directory